tests/unit/handlers/test_handlers.py

```diff
@@ -165,8 +166,60 @@ def test_setup_logging_excludes(self):
         self.assertNotIn(handler, excluded_logger.handlers)
         self.assertFalse(excluded_logger.propagate)
 
+    def test_remove_handlers_gcf(self):
```

Comment:
I recommend using `mock.patch.dict` for temporarily modifying `os.environ`. It will handle clean up as well. https://docs.python.org/3/library/unittest.mock.html#patch-dict

Reply:
Thanks, that looks much better